tests/integration/cmd/devfileregistry_test.go

```diff
@@ -38,16 +39,45 @@ const (
 
 //SynchronizedBeforeSuite blocks is executed before run all test suites
 var _ = ginkgo.SynchronizedBeforeSuite(func() []byte {
+	var err error
+
 	fmt.Println("Starting to setup objects before run ginkgo suite")
 	registry := os.Getenv("REGISTRY")
 	if registry == "" {
 		registry = "https://registry.devfile.io"
 	}
 	config.Registry = registry
 	config.RegistryList = registry + "," + "https://registry.stage.devfile.io"
-	os.Setenv("REGISTRY_LIST", config.RegistryList)
+	err = os.Setenv("REGISTRY_LIST", config.RegistryList)
+	if err != nil {
+		fmt.Println(err)
+		panic(err)
+	}
+
+	// If the registry is the test one, set IsTestRegistry variable to run extra test cases
+	if isTestRegistry, isSet := os.LookupEnv("IS_TEST_REGISTRY"); isSet {
+		config.IsTestRegistry, err = strconv.ParseBool(isTestRegistry)
+		if err != nil {
+			fmt.Println(err)
+			panic(err)
+		}
+	} else {
+		config.IsTestRegistry = false
```

Comment:
I don't think we need the else statement because unset booleans are false by default